internal/imports/imports.go

```diff
@@ -0,0 +1,124 @@
+// Copyright ©2018 The Gonum Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style
+// license that can be found in the LICENSE file.
+
+// Package imports provides an API to check whether Gonum code does
+// not import deprecated packages.
+package imports // import "gonum.org/v1/tools/internal/imports"
+
+import (
+	"fmt"
+	"go/parser"
+	"go/token"
+	"io/ioutil"
+	"os"
+	"path/filepath"
+	"regexp"
+	"strings"
+)
+
+// CheckBlacklisted analyzes all Go files under dir for deprecated and
+// blacklisted imports.
+// If Check encounters multiple files importing deprecated imports, the
```

Comment:
s/Check/CheckBlacklisted/

Can you add a TODO to handle multiple errors, perhaps with a limit as gc does.

Reply:
done